sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -36,6 +42,18 @@ export interface EventData {
     };
 }
 
+// @public
+export class EventDataBatch {
+    // Warning: (ae-forgotten-export) The symbol "ConnectionContext" needs to be exported by the entry point index.d.ts
+    // 
+    // @internal
+    constructor(context: ConnectionContext, maxSizeInBytes: number, partitionKey?: string);
+    encodedBatchMessage: Buffer | undefined;
```

Comment:
How about just `batchMessage` or `eventBatch`? Since this is a public facing api, am wondering why the user should care if the message is encoded or not